commands.txt

```diff
@@ -1,4 +1,4 @@
-if [ ! -z "$SD_PULL_REQUEST" ]; then export SD_SONAR_OPTS="${SD_SONAR_OPTS} -Dsonar.pullrequest.key=$SD_PULL_REQUEST -Dsonar.pullrequest.branch=${PR_BRANCH_NAME#*/} -Dsonar.pullrequest.base=${PR_BASE_BRANCH_NAME}"; fi
+if [ ! -z "$SD_PULL_REQUEST" ] && [ "$SD_SONAR_ENTERPRISE" == true ]; then export SD_SONAR_OPTS="${SD_SONAR_OPTS} -Dsonar.pullrequest.key=$SD_PULL_REQUEST -Dsonar.pullrequest.branch=${PR_BRANCH_NAME#*/} -Dsonar.pullrequest.base=${PR_BASE_BRANCH_NAME}"; fi
```

Comment:
this should be `[ "$SD_SONAR_ENTERPRISE" = true ]` for POSIX compatibility

Reply:
updated